app.rb

```diff
@@ -0,0 +1,107 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_all_books
+    puts @books
+  end
+
+  def list_all_people
+    @people.each do |person|
+      puts "[#{person.class}] ID: #{person.id}, Name: #{person.name}, Age: #{person.age}"
+    end
+  end
+
+  def create_student
+    print 'Age '
+    age = gets.chomp
+
+    print 'Name '
+    name = gets.chomp
+
+    print 'Has parent permission? [Y/N]: '
+    has_permission = gets.chomp
+    parent_permission = has_permission.include? 'Yy'
+
+    @people << Student.new(age, name, parent_permission)
+    puts 'Person created successfully'
+  end
+
+  def create_teacher
+    print 'Age '
+    age = gets.chomp
+
+    print 'Name '
+    name = gets.chomp
+
+    print 'Specialization '
+    specialization = gets.chomp
+
+    @people << Teacher.new(age, specialization, name)
+    puts 'Person created successfully'
+  end
+
+  def create_person
+    print 'Do you want to create a student(1) a teacher(2)? [Input tNhe number]:'
```

Comment:
- [ ] [OPTIONAL] Please fix the typo, i think you wanted to write `the` not `tNhe`

![image](https://user-images.githubusercontent.com/75126481/160101010-85c82366-e1d6-42b3-a280-1069ed2f47e3.png)